### PR TITLE
Add files

### PR DESCRIPTION
**Pull request by Pasionpreciouskaye**
please review my files, thankyou

### CODE REVIEW

File: pasion_preciouskaye/example1.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Cat</title>
+</head>
+<body>
+      Hi!
+
```
Comment (ternalapriljoy) [inline pasion_preciouskaye/example1.html:10]: Kindly remove this space.

Comment (Pasionpreciouskaye) [reply pasion_preciouskaye/example1.html:10]: thankyou

File: pasion_preciouskaye/index.html [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+      />
+      <img
+        src="https://yt3.googleusercontent.com/HskYwAGlO-VcPpnUNJcfTKBbFrj2LNrRbrm1Ht5cuHhc621f2Tgzf-deKf3uL0qMY8_KRApMoZg=s900-c-k-c0x00ffffff-no-rj"
+      />
+      <img
+        src="https://i.pinimg.com/236x/b7/91/a9/b791a92b8fc7819738019516d168ed25.jpg"
+      />
+      <img
+      src="https://i.pinimg.com/736x/69/5f/2f/695f2f95029f73e501cf28263b5aa492.jpg"
+    />
```
Comment (ternalapriljoy) [inline pasion_preciouskaye/index.html:102]: https://github.com/PUP-BSIT/exercise-13-tech_nx/blob/c0e8cf05d5790a8696f6421541e2f159ccaffc67/pasion_preciouskaye/index.html#L87

https://github.com/PUP-BSIT/exercise-13-tech_nx/blob/c0e8cf05d5790a8696f6421541e2f159ccaffc67/pasion_preciouskaye/index.html#L90

https://github.com/PUP-BSIT/exercise-13-tech_nx/blob/c0e8cf05d5790a8696f6421541e2f159ccaffc67/pasion_preciouskaye/index.html#L93

https://github.com/PUP-BSIT/exercise-13-tech_nx/blob/c0e8cf05d5790a8696f6421541e2f159ccaffc67/pasion_preciouskaye/index.html#L96

https://github.com/PUP-BSIT/exercise-13-tech_nx/blob/c0e8cf05d5790a8696f6421541e2f159ccaffc67/pasion_preciouskaye/index.html#L99

https://github.com/PUP-BSIT/exercise-13-tech_nx/blob/c0e8cf05d5790a8696f6421541e2f159ccaffc67/pasion_preciouskaye/index.html#L102


You may just attach the </ at the end of the line so it won't have an unnecessary excess line of code.

https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit?fbclid=IwZXh0bgNhZW0CMTAAAR04rIg_SBkft7ED5muFsxm9FUkijQq-aweBdgT21-tlg-UpXKtfkO4Nb9s_aem_AanCGQNw3v8o-ZJK1bprYBSLNcABosgmI-0dnlGiU_GSgrlxM4drKiB_S99jcEItxjMmHfHXZ64dF_L3NbTavA6s#slide=id.g24da8de92d7_0_0

File: pasion_preciouskaye/index.html [hunk 3]
```diff
@@ -0,0 +1,186 @@
...
+          <textarea name="ycomment" id="ycomment"
+          cols="35" rows="10"></textarea>
+        </div>
+    
+        <div>
+          <button type="submit" id="button_comment" disabled>Submit</button>
+        </div>
+      </form>
+      
+      <script src="./script.js"></script>
```
Comment (ternalapriljoy) [inline pasion_preciouskaye/index.html:141]: https://github.com/PUP-BSIT/exercise-13-tech_nx/blob/c0e8cf05d5790a8696f6421541e2f159ccaffc67/pasion_preciouskaye/index.html#L141

It's quite unusual to see this here. Please put this line of code before the closing tag for body.

File: pasion_preciouskaye/style.css [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+  grid-area: message;
+}
+
+.submit {
+  grid-area: submit;
+  padding: 10px;
+  text-align: center;
+}
+
+
```
Comment (ternalapriljoy) [inline pasion_preciouskaye/style.css:131]: https://github.com/PUP-BSIT/exercise-13-tech_nx/blob/c0e8cf05d5790a8696f6421541e2f159ccaffc67/pasion_preciouskaye/style.css#L131

Excess space. Please remove.

File: pasion_preciouskaye/script.js [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+    nameInput.addEventListener('input', checkFormValidity);
+    commentInput.addEventListener('input', checkFormValidity);
+  
+    function checkFormValidity() {
+      if (nameInput.value.trim() !== '' && commentInput.value.trim() !== '') {
+        submitButton.disabled = false;
+      } else {
+        submitButton.disabled = true;
+      }
+    }
```
Comment (ternalapriljoy) [inline pasion_preciouskaye/script.js:32]: Kindly review this code as the submit button is still disabled even if the name and comment have inputs already.

If possible, I think you can have this code written in guard clause. 

https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit?fbclid=IwZXh0bgNhZW0CMTAAAR04rIg_SBkft7ED5muFsxm9FUkijQq-aweBdgT21-tlg-UpXKtfkO4Nb9s_aem_AanCGQNw3v8o-ZJK1bprYBSLNcABosgmI-0dnlGiU_GSgrlxM4drKiB_S99jcEItxjMmHfHXZ64dF_L3NbTavA6s#slide=id.g25d183194fb_0_0

**Review by ternalapriljoy — CHANGES_REQUESTED** [review]
Please kindly check your buttons if working or not before commiting your codes. Overall, your output has a well-written code. It's readable. :)

**Pull request merged**